cumulus_library/studies/core/core_templates/core_utils.jinja

```diff
@@ -90,22 +90,22 @@ targets is assumed to be a list of tuples of one of the following format:
         {%- for col in targets %}
         {%- if col is not string and col|length ==4%}
         {%- if schema[table][col[0]][col[1]] %}
-        date_trunc('{{ col[3] }}', date(from_iso8601_timestamp({{ alias }}.{{ col[0] }}.{{ col[1] }}))) 
+        date_trunc('{{ col[3] }}', date(from_iso8601_timestamp({{ alias }}."{{ col[0] }}"."{{ col[1] }}")))
```

Comment:
This came up because now that the regression test actually sees the `period.end` field, it tries to access it. But `end` is a keyword that can't just be injected like that.

So I took the safest but ugliest approach - quoting each arbitrary field access.
- A less safe but prettier option is just quoting the last field access. That's where trickier words like `end` and `date` are likely to show up.
- The least safe but prettiest option is to have the caller of this quote it. Like `('period', '"end"', 'period_end_day', 'day'),` when calling `truncate_date_cols`